eksctl/README.md

```diff
@@ -0,0 +1,51 @@
+---
+title: eksctl
+homepage: https://github.com/weaveworks/eksctl
+tagline: |
+  The official CLI for Amazon EKS
```

Comment:
Is this actually endorsed by Amazon? I think you may mean "original" or "best", but I'm pretty sure this isn't the "official" CLI. That's written in Python and described here: https://docs.aws.amazon.com/cli/latest/reference/eks/index.html

Or am I missing something?